docs/conf.py

```diff
@@ -84,3 +100,15 @@
 sitemap_locales = [None]
 # We're hard-coding stable here since that's what we want Google to point to.
 sitemap_url_scheme = "{link}"
+
+# -- For sphinx_reredirects ------------------------------------------------
+
+redirects = {
+    "getting-started": "user-guide",
+    "faq": "user-guide/faq",
+    "configuration": "user-guide/configuration",
+    "libmamba-vs-classic": "user-guide/libmamba-vs-classic",
+    "more-resources": "user-guide/more-resources",
+    "performance": "user-guide/performance",
+    "subcommands": "user-guide/subcommands",
+}
```

Comment:
Something was wrong with this section: https://github.com/conda/conda-libmamba-solver/issues/374